anomaly_agent/phase_aware_handler.py

```
@@ -393,3 +397,45 @@ def get_summary_stats(self) -> Dict[str, Any]:
+            })
+            
+            report_generator.record_anomaly(
+                anomaly_type=decision['anomaly_type'],
+                severity=severity,
+                confidence=decision.get('detection_confidence', 0.0),
+                mission_phase=decision['mission_phase'],
+                telemetry_data=telemetry_data,
+                explanation=decision.get('explanation')
+            )
+            
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)